modules/mobile/app/serializers/mobile/v0/user_serializer.rb

```diff
@@ -94,12 +94,17 @@ def profile
 
       def authorized_services
         auth_services = SERVICE_DICTIONARY.filter { |_k, policies| authorized_for_service(policies) }.keys
-        if auth_services.include?(:directDepositBenefits) && user.authorize(:ppiu, :access_update?)
-          auth_services.push(:directDepositBenefitsUpdate)
-        end
+        auth_services.push(:directDepositBenefitsUpdate) if can_access_direct_deposit_benefits?(auth_services)
         auth_services
       end
 
+      def can_access_direct_deposit_benefits?(auth_services)
+        auth_services.include?(:directDepositBenefits) && user.authorize(:ppiu, :access_update?)
+      rescue EVSS::ErrorMiddleware::EVSSError
+        # this is a temporary patch for upstream issues
+        false
```

Comment:
I would add some logging here or a metric so we can see how often this is happening more easily

Reply:
I thought about it but I assumed it would just output: `[{"key"=>"paymenthistory.partner.service.failed", "severity"=>"ERROR", "text"=>"Call to partner getPaymentHistory failed"}]`
I'm not certain though, so I'll add something and we'll see what happens.